src/backend/gporca/libgpopt/include/gpopt/mdcache/CMDCache.h

```diff
@@ -86,6 +93,20 @@ class CMDCache
 		return m_pcache;
 	}
 
+	// mark cache as transient
+	static void
+	MarkContainTransientRelation(uint32_t xmin)
```

Comment:
```suggestion
	SetTransientXmin()
```

Reply:
i think that `MarkContainTransientRelation` describes better what it does.